pkg/report/linux.go

```diff
@@ -433,6 +433,16 @@ var linuxStackKeywords = []*regexp.Regexp{
 	regexp.MustCompile(`[^k] backtrace:`),
 }
 
+func stacktraceRe(frameBlacklist []string) string {
```

Comment:
If you change this to `stacktraceRe(frameBlacklist ...string)`, it will allow nicer call syntax. We will be able to use `stacktraceRe("stack", "lock", "IRQ")` iInstead of `stacktraceRe([]string{"stack", "lock", "IRQ"})`.

Reply:
Didn't know that syntax, done.